static/js/chart/draw_scatter.ts

```diff
@@ -853,13 +849,18 @@ export function drawScatter(
   const xMinMax = d3.extent(Object.values(points), (point) => point.xVal);
   const yMinMax = d3.extent(Object.values(points), (point) => point.yVal);
 
-  let height = properties.height - MARGINS.top - MARGINS.bottom;
+  let marginTop = MARGINS.top;
```

Comment:
Discussion: should we rename MARGINS to MARGIN to match the naming used in the other files?

Reply:
i like that, done